.github/workflows/update-git-blame-ignore-revs.yml

```diff
@@ -0,0 +1,31 @@
+name: 🙈 Update git-blame-ignore-revs
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - release-*
+
```

Comment:
explicit "permissions:" at the workflow and job level would be best to make it obvious which ones are needed